### PR TITLE
Add basic percy setup

### PR DESCRIPTION
**Pull request by alxnddr**
### Description

This PR adds a basic visual testing setup using [Percy](https://percy.io/). Since we want to end up having our CI in Github Actions I decided not to run it in CircleCI.

In order to test this PR, you need to obtain a `PERCY_TOKEN` and you can do that by creating a new Percy organization or by pinging me so that I will add you to my test organization and share the credentials. Then, you need to fork the Metabase, add `PERCY_TOKEN` to your repo secrets and merge these changes to your master branch.

The PR contains 2 workflows:
- `.github/workflows/percy.yml` runs on `push` to `master` and `release-` branches to create baseline screenshots
- `.github/workflows/percy-issue-comment.yml` runs on a PR comment with text `@metabase-bot run visual tests`

Note: `percy-issue-comment` workflow is always used from the default branch so unless the changes are in `master` it is not possible to use `@metabase-bot run visual tests` command.

#### 1. Engineer wants to run Percy tests on a PR and posts a comment `@metabase-bot run visual tests`
Screen Shot 2021-07-23 at 15 35 53

It triggers a workflow that is visible in the repo Actions tab 
Screen Shot 2021-07-23 at 15 39 21

#### 2. When there are some visual changes it shows a failed Percy check in the PR
Screen Shot 2021-07-23 at 17 20 26

#### 3. Once an engineer reviews and approves the changes the PR check becomes green
Screen Shot 2021-07-23 at 17 20 45

Screen Shot 2021-07-23 at 17 21 05

### TODO (probably, it can be done as a follow-up)
- Investigate missing Lato font issue
- Ensure no flakiness
- Add way more specs

### CODE REVIEW

File: .github/workflows/.percy.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+name: Percy
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "docs/**"
+      - "frontend/test/**"
```
Comment (ariya) [inline .github/workflows/.percy.yml:8]: The visual tests will be in this directory, right? Should it be in the ignore list?

**Review by gusaiani — APPROVED** [review]
Seems to work as expected.

There are open ends still, like the diff conflict, happy to review again if needed but here's my approval.

Great stuff!

File: .github/workflows/.percy.yml (hunk 1 above)
Comment (alxnddr) [reply .github/workflows/.percy.yml]: Thanks for catching, fixed that

**Comment by alxnddr** [comment]
Ariya raised a point that because of a slow build due to using Github Actions until we migrated from Circle CI it might be annoying to see "Some checks haven’t completed yet" on every PR even though the Percy check is not required. Also, it requires changing everyone's workflow to ensure PRs are in draft mode during the development/review fixes. Because of that, I changed the way to trigger the Percy job: just write `@metabase-bot run visual tests` in a PR comment which will trigger the Percy job in a PR. Once we add more tests, migrate CI to Github Actions, speed up the build, check how much it costs for us, we will be able to run it on every commit

**Comment by gusaiani** [comment]
@metabase-bot run visual tests

**Review by gusaiani — APPROVED** [review]
It's looking good to me.

Ariya's suggestions made a whole lot of sense to have in this initial PR.

So...if others want to take a look, great, otherwise we can tweak in future PRs.

**Comment by kulyk** [comment]
Hey @alxnddr, great work here! Just curious how the expected workflow for Percy would look in the nearest future? Let's say I open a PR with some visual changes, should I leave a "run visual tests" comment on the PR when it's ready for review?

**Review by kulyk — APPROVED** [review]
🔥 🔥 🔥

**Comment by alxnddr** [comment]
@kulyk yes, you're right. It is definitely worth checking before the review that nothing is broken, however, you can also use it in the middle of implementation just to check how the changes affected the app

**Pull request merged**